frontend/src/pages/ActivityReport/Pages/Review/ReviewItem.js

```diff
@@ -94,6 +76,10 @@ const ReviewItem = ({
     values.sort();
   }
 
+  if (values.length === 0 || values[0] === undefined) {
```

Comment:
Can probably just check `if (values.length)`. The only case for going into the first item in the array might be if you have seen cases where this can happen.

Reply:
I remember I did see an instance where it was creating a 1-length array but the initial item was undefined.